src/services/auth/plugins/password/service.ts

```diff
@@ -85,7 +85,7 @@ export class MemberPasswordService {
     if (!id) {
       return;
     }
-    await this.redis.del(uuid);
+    await this.redis.del(`${REDIS_PREFIX}${uuid}`);
```

Comment:
Wouldn't it be clearer if you have a private fn to build the prefix? So you can reuse at get, post and delete.